cargo-insta/src/container.rs

```diff
@@ -213,22 +217,28 @@ impl SnapshotContainer {
             for snapshot in self.snapshots.iter() {
                 match snapshot.op {
                     Operation::Accept => {
-                        let snapshot = Snapshot::from_file(&self.pending_path).map_err(|e| {
```

Comment:
I'm not sure about this. But it seemed to me that loading the snapshot again here is redundant. It seems this was introduced in #218 originally (the code was still in `cargo.rs` then). If we just use the snapshot from `snapshot.new` here it should also mitigate the problem referenced in #516. The only reason I can see for loading it again separately would be to support updating pending snapshots while reviewing, but it seems a lot of other things could go wrong there as well.

But it's totally possible, that I'm just missing something.

Reply:
Yes, very much agree, great spot.

(though I also may be missing something...)